docs/report-entries.adoc

```diff
@@ -1,5 +1,6 @@
 :page-title: Publishing Report Entries
 :page-description: Extends JUnit Jupiter with `@ReportEntry`, which publishes report entries using `TestReporter`.
+:snippetsource: ../src/demo/java/org/junitpioneer/jupiter/ReportEntryExtensionTests.java
```

Comment:
This attribute points at the wrong file.

That shows that this way of referencing is brittle. Would be nice to verify during a build that the docs actually "work" but then we'd need to include some asciidoc in the build, which.... ugh. Let's ignore that if it doesn't cause too many problems.

Reply:
What do you mean? It works on _my_ machine! 😱 

Yeah, that's annoying, I'll fix it.